python/src/dist_bundle3_module.i

```diff
@@ -47,6 +47,7 @@
 %include Uniform.i
 %include UniformFactory.i
 %include UniformMuSigma.i
+%include UniformOverMesh.i
```

Comment:
remember new classes must go to the experimental module

Reply:
It was what I had in mind, but forgot it at the last minute :-(